### DISCUSSION TITLE
Add server information to the left panel of android app

### DISCUSSION THREAD

**Discussion by amitkshatriya01** [immich-app/immich#3670] [General]
![Screenshot_20230813_232230_01](https://github.com/immich-app/immich/assets/46089694/bc38b910-cede-4d52-a3c4-03cb9953529e)

Setup immich today and got it working on my phone, I wanted to setup my wife's phone as well but forgot the IP of my homeserver. I thought it should be somewhere accessible on my android app so tried looking for it but couldn't find it. I think adding the server information on the pull out panel should have helped. If this already exists somewhere else then do ignore.

**Comment by alextran1502** [comment]
It can be viewed in the panel when clicking on the top left icon from the main timeline

**Reply by amitkshatriya01** [reply]
This icon?
![Screenshot_20230813_235054](https://github.com/immich-app/immich/assets/46089694/997be86d-6215-4ef2-927f-221bfc55449f)

**Comment by alextran1502** [comment]
Yes

**Reply by jrasm91** [reply]
Does it include the server url? I don't see it on mine

**Reply by amitkshatriya01** [reply]
Clicking this icon opens the side panel but that does not have server URL.

**Reply by jrasm91** [reply]
I think it would be nice to show the server url along with the server version.
